tsconfig.json

```diff
@@ -7,6 +7,7 @@
         "sourceMap": true,
         "incremental": true,
         "strict": true,
+        "esModuleInterop": true,
```

Comment:
This is required to make the newer version of httpsproxyagent build. The newer version is required to make it package properly.

Reply:
you'd think that by putting `"target": "ES6",`, this would be on implicitly